sdk/src/wallet/core/mod.rs

```diff
@@ -299,9 +299,12 @@ impl WalletLedger {
     }
 
     // Returns the first possible unexpired block issuer Account id, which can be an implicit account.
-    pub fn first_block_issuer_account_id(&self, current_slot: SlotIndex) -> Option<AccountId> {
+    pub fn first_block_issuer_account_id(&self, current_slot: SlotIndex, network_id: u64) -> Option<AccountId> {
```

Comment:
Seems annoying :(

Reply:
Current slot is also annoying, but it's only on the WalletLedger, from the Wallet you get it without providing any params, so I don't think it's that bad

Reply:
Do we even need the check? I guess there are plenty of places where we could do this and don't

Reply:
Don't see where else this would be an issue, here it was an issue for me because I changed the node and then it selected the account from another network (in the input selection we filter this out, but for the issuer we didn't)

Reply:
Ok